ci/mkpipeline.py

```diff
@@ -34,7 +34,7 @@
 from materialize import mzbuild, spawn
 from materialize.mzcompose.composition import Composition
 
-from ..deploy.deploy_util import rust_version
+from .deploy.deploy_util import rust_version
```

Comment:
Nit: Why are we using relative imports here? I had thought the linter does not allow that...

Reply:
I don't know to be honest, also confused me a bit. Didn't want to rock the boat ;)

Reply:
That's ok. Ignore it.